litmus/director/openebs-install/test.yml

```diff
@@ -27,9 +27,31 @@
     - name: Get password
       shell: cat /etc/secret-volume/password
       register: password
-    ## Fetch Project deatils
 
     - block:
+
+        ## Checking if openebs is already installed in the cluster
+        - name: Check if openebs is already installed
+          shell: kubectl get deploy --all-namespaces | grep maya-apiserver | awk '{print $1}' | wc -l
+          args:
+            executable: /bin/bash
+          register: count_maya_api
+
+        ## If openebs is installed in the cluster then get the namespace
+        - name: Getting the namespace where openebs is installed 
+          shell: kubectl get deploy --all-namespaces | grep maya-apiserver | awk '{print $1}'
+          args:
+            executable: /bin/bash
+          register: openebs_namespace
+          when: "count_maya_api.stdout == '1'"
+
+        ## Delete openebs 
```

Comment:
if you directly delete the namespace then its a problem, there is a proper flow to unintall openebs: https://docs.openebs.io/docs/next/uninstall.html 
for ex, in user-cluster-connect an application is running that mean pvs are there with openebs storage class.